cmd/admin/main.go

```diff
@@ -163,7 +158,7 @@ func main() {
 	api_admin.HandlerFromMux(
 		api_admin.NewStrictHandlerWithOptions(
 			api_admin.NewServer(cfg, identityService, claimsService, schemaService, publisher, packageManager, serverHealth),
-			middlewares(ctx, cfg.Admin.HTTPAdminAuth),
+			middlewares(ctx, cfg.ApiUI.HTTPAPIUIAuth),
```

Comment:
Recommend that all instances of API are uppercased (Api -> API, api -> API etc.)